diff --git a/Inc/config.h b/Inc/config.h
index f363189..618376b 100644
--- a/Inc/config.h
+++ b/Inc/config.h
@@ -9,6 +9,8 @@
 // Keil uVision: select desired variant from the Target drop down menu (to the right of the Load button)
 // Ubuntu: define the desired build variant here if you want to use make in console
 // or use VARIANT environment variable for example like "make -e VARIANT=VARIANT_NUNCHUK". Select only one at a time.
+#define VARIANT_USART       // Variant for Serial control via USART3 input
+
 #if !defined(PLATFORMIO)
   //#define VARIANT_ADC         // Variant for control via ADC input
   //#define VARIANT_USART       // Variant for Serial control via USART3 input
@@ -249,8 +251,8 @@
 */
 
 // #define DEBUG_SERIAL_USART2          // left sensor board cable, disable if ADC or PPM is used!
-#define DEBUG_SERIAL_USART3          // right sensor board cable, disable if I2C (nunchuk or lcd) is used!
-#define DEBUG_SERIAL_PROTOCOL        // uncomment this to send user commands to the board, change parameters and print specific signals (see comms.c for the user commands)
+//#define DEBUG_SERIAL_USART3          // right sensor board cable, disable if I2C (nunchuk or lcd) is used!
+//#define DEBUG_SERIAL_PROTOCOL        // uncomment this to send user commands to the board, change parameters and print specific signals (see comms.c for the user commands)
 // ########################### END OF DEBUG SERIAL ############################
 
 
@@ -310,8 +312,8 @@
 // ############################ VARIANT_USART SETTINGS ############################
 #ifdef VARIANT_USART
 
-#define CONTROL_SERIAL_USART2 0
-#define FEEDBACK_SERIAL_USART2 
+#define CONTROL_SERIAL_USART3 0
+#define FEEDBACK_SERIAL_USART3 
 
   // #define SIDEBOARD_SERIAL_USART2 0
   // #define CONTROL_SERIAL_USART2  0    // left sensor board cable, disable if ADC or PPM is used! For Arduino control check the hoverSerial.ino
